src/components/Feed/Edit.jsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import FormFeed from "./Form";
+import { useState, useEffect } from "react";
+import { useParams, Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+
+const EditFeed = () => {
+    const [feed, setFeed] = useState({})
+    const navigate = useNavigate()
+    const params = useParams()
+    const [media, setMedia] = useState([])
+    const [loading, setLoading] = useState(false)
+
+    useEffect(() => {
+        const id = params.id
+        const url = `http://127.0.0.1:3000/api/v1/feeds/${id}`
+        axios.get(url)
+            .then(resp => {
+                setFeed(resp.data)
+            })
+            .catch(resp => console.log(resp))
+    }, [])
+
+    const handleChange = (e) => {
+        e.preventDefault()
+        setFeed(Object.assign({}, feed, { [e.target.name]: e.target.value }))
+    }
+
+    const handleMediaChange = (e) => {
+        setMedia([...e.target.files])
+    }
+
+    const handleSubmit = (e) => {
+        e.preventDefault()
+        setLoading(true)
+        const id = params.id
+        const formData = new FormData()
+
+        for (var key in feed) {
+            formData.append(`feed[${key}]`, feed[key]);
+        }
+        formData.delete('feed[media]')
+        media.forEach((item, index) => (
+            formData.append('feed[media][]', item)
+        ))
+
+        const url = `http://127.0.0.1:3000/api/v1/feeds/${id}`
```

Comment:
```suggestion
        const url = `${import.meta.env.BASE_URL}/api/v1/feeds/${id}`
```
Move localhost:3000 address to .env